section6Arrays/mergeArrays/main.go

```diff
@@ -0,0 +1,13 @@
+package mergeArrays
+
+import "sort"
+
+func mergeArrays(slc1, slc2 []int) []int {
+	var mergeSlice []int
+
+	mergeSlice = append(mergeSlice, slc1...)
+	mergeSlice = append(mergeSlice, slc2...)
```

Comment:
I think you should write your own merge logic rather than using library method

Reply:
fixed